source/_integrations/hue_ble.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: Hue BLE
+description: Instructions on how to setup Philips Hue Bluetooth lights within Home Assistant.
+ha_category:
+  - Light
+ha_release: 2024.7
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@flip-dots'
+ha_domain: hue_ble
+ha_platforms:
+  - light
+ha_bluetooth: true
+ha_integration_type: integration
+---
+
+The **Hue BLE** {% term integration %} allows you to control your Philips Hue Bluetooth lights with Home Assistant.
+
+## Initial setup
+
+<div class='note'>
+
+Before trying to connect your light(s) to Home Assistant, you must put the light(s) into pairing mode in the Hue app ([Android](https://play.google.com/store/apps/details?id=com.philips.lighting.hue2), [iOS](https://apps.apple.com/us/app/philips-hue/id1055281310)).
+
+**Settings** > **Voice Assistants** > **Amazon Alexa** or **Google Home** > **Make Discoverable**.
+
+When the light is in pairing mode you may proceed to connect to it with Home Assistant. Discovered lights can be found in the discovery section or in the integration setup screen.
+
+</div>
```

Comment:
```suggestion
{% endnote %}
```
we have these liquid [tags to highlight text](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#text-boxes) in text boxes.